src/elements/element.bar.js

```diff
@@ -83,16 +83,19 @@ function parseBorderWidth(bar, maxW, maxH) {
 }
 
 function parseBorderRadius(bar, maxW, maxH) {
+  const {endOfStack, float} = bar.getProps(['endOfStack', 'float']);
   const value = bar.options.borderRadius;
   const o = toTRBLCorners(value);
   const maxR = Math.min(maxW, maxH);
   const skip = parseBorderSkipped(bar);
 
+  const enableBorder = endOfStack || float;
```

Comment:
I was thinking about making this more opt-in. If `bar.options.borderRadius` is a number, we consider `endOfStack` and `float`. If it's an object, we just apply it as specified to enable the most configuration. Thoughts?